source/crud/tests/README.rst

```diff
@@ -58,6 +60,9 @@ Each YAML file has the following keys:
     - ``name``: The name of the operation as defined in the specification. The
       name `db-aggregate` refers to database-level aggregation.
 
+    - ``object``: The name of the object to perform the operation on. Can be
+      "database" or "collection". Defaults to "collection" if undefined.
+
     - ``database_name`` (optional): The name of the database against which to execute the operation.
```

Comment:
I think this can be removed as `database_name` has moved to a top level field.

Reply:
The [tests](https://github.com/mongodb/specifications/blob/master/source/crud/tests/db/db-aggregate.yml) don't appear to be using the top-level `database_name` field yet. These tests are a bit unique in that they're running `$currentOp`, which is outside the scope of a single database. While "admin" is the target database on which the aggregation is executed, it's not technically the database under test (in the same way a collection would be in other tests).

That said, I do agree we should use one of the other. A top-level `database_name` key would be consistent with other spec tests.

Reply:
@rozza: Good to resolve?

Reply:
All good - thanks :+1: